src/framework/mpas_stream_manager.F

```diff
@@ -5159,6 +5197,11 @@ subroutine postwrite_reindex(allFields, streamFields) !{{{
 
         integer :: i, j, threadNum
 
+        character (len=StrKIND), pointer :: packages
+        logical :: active_field
+        integer :: err_level
```

Comment:
Looking through these code changes again, I'm not sure why I've added these variable declarations here.